MacPass/MPPasswordInputController.m

```diff
@@ -185,31 +187,20 @@ - (void)_showError:(NSError *)error {
 - (NSTouchBar *)makeTouchBar {
   NSTouchBar *touchBar = [[NSTouchBar alloc] init];
   touchBar.delegate = self;
-  touchBar.defaultItemIdentifiers = @[touchBarShowPasswordIdentifier, touchBarChooseKeyfileIdentifier, NSTouchBarItemIdentifierFlexibleSpace,touchBarUnlockIdentifier];
+  touchBar.customizationIdentifier = touchBarIdentifier;
+  NSArray<NSTouchBarItemIdentifier> *defaultItemIdentifiers = @[touchBarShowPasswordIdentifier, touchBarChooseKeyfileIdentifier, NSTouchBarItemIdentifierFlexibleSpace,touchBarUnlockIdentifier];
+  touchBar.defaultItemIdentifiers = defaultItemIdentifiers;
+  touchBar.customizationAllowedItemIdentifiers = defaultItemIdentifiers;
   return touchBar;
 }
 
 - (NSTouchBarItem *)touchBar:(NSTouchBar *)touchBar makeItemForIdentifier:(NSTouchBarItemIdentifier)identifier  API_AVAILABLE(macos(10.12.2)) {
   if (identifier == touchBarChooseKeyfileIdentifier) {
-    NSCustomTouchBarItem *item = [[NSCustomTouchBarItem alloc] initWithIdentifier:touchBarChooseKeyfileIdentifier];
-    NSImage *image = [NSImage imageNamed:NSImageNameTouchBarFolderTemplate];
-    NSButton *button = [NSButton buttonWithTitle:@"Choose Keyfile" image:image target:self.keyPathControl action:@selector(showOpenPanel:)];
-    item.view = button;
-    return item;
+    return [MPTouchBarButtonCreator touchBarButtonWithTitleAndImage:@"Choose Keyfile" identifier:touchBarChooseKeyfileIdentifier image:[NSImage imageNamed:NSImageNameTouchBarFolderTemplate] target:self.keyPathControl selector:@selector(showOpenPanel:) customizationLabel:@"Choose Keyfile"];
```

Comment:
For user-facing text you should use localised strings. In this case:
```
[MPTouchBarButtonCreator touchBarButtonWithTitleAndImage:NSLocalizedString(@"TOUCHBAR_BUTTON_CHOOSE_KEYFILE", "A usefull comment") identifier:touchBarChooseKeyfileIdentifier image:[NSImage imageNamed:NSImageNameTouchBarFolderTemplate] target:self.keyPathControl selector:@selector(showOpenPanel:) customizationLabel:NSLocalizedString(@"OPEN_PANEL_CHOOSE_KEYFILE", "A usefull comment")];
```

Do not pin me down on the exact keys :). I think you get the idea.